WordPress/Classes/ViewRelated/Site Creation/WebAddressStep/WebAddressWizardContent.swift

```diff
@@ -0,0 +1,119 @@
+import UIKit
+
+final class WebAddressWizardContent: UIViewController {
```

Comment:
I thought there might be an opportunity to adapt and/or reuse [`DomainSuggestionsTableViewController`](https://github.com/wordpress-mobile/WordPress-iOS/blob/2d4089e965c7db2c3f458873278d3e1b17bf558d/WordPress/Classes/ViewRelated/NUX/DomainSuggestionsTableViewController.swift#L14). 

I think we could inject differing title views for (a) "legacy" & (b) "enhanced" site creation flows, and then extend the class to conform to our `Wizard` / `WizardStep` protocols.

Thoughts?

Reply:
I am not sure, to be honest, at least not right now. I think these three view controllers, as they are, can be simplified into one. I was waiting to have some clarity on how the Site info is going to look before doing so. Shall we revisit this after the three XXXWizardContent are simplified into one?